interview-questions.md

```diff
@@ -7,49 +7,49 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer:I'm not sure if this was some sort of trick question or not because from what I can remember is that they can either be accessors and mutators.
 
```

Comment:
It is a bit of a trick since they can be both, but as a developer you have control over when it is an accessor and when it is a mutator.